Mustachio/ParsingOptions.cs

```diff
@@ -1,10 +1,23 @@
 ﻿namespace Mustachio
 {
-    internal class ParsingOptions
+    public class ParsingOptions
     {
         /// <summary>
         /// If this is true, all values will be rendered without being HTML-encoded. (regardless of using {{{ }}} or {{ }} syntax)
+        /// In some cases, content should not be escaped (such as when rendering text bodies and subjects in emails). 
+        /// By default, we use content escaping, but this parameter allows it to be disabled.
         /// </summary>
-        internal bool DisableContentSafety { get; set; }
+        public bool DisableContentSafety { get; set; } = false;
+
+        /// <summary>
+        /// The source name for this template. Will be used in error reporting to identify the location of parsing errors.
+        /// </summary>
+        public string SourceName { get; set; } = "Base";
```

Comment:
@vladsandu this should be `null` -- null means "I don't know the answer, could be anything.".